mods/projectHELLFIRE/mod.xml

```diff
@@ -35,3 +35,5 @@ ffnx conversion and extra textures by Slimebucket</Description>
 
 
 
+
+

```

Comment:
Can we please also make sure we're not adding 25 new lines per file? :) Thank you